src/Themes/HackathonTheme/Views/OrchardCore.Users/ResetPassword/ResetPassword.cshtml

```diff
@@ -2,46 +2,49 @@
-      </div>
-      <div class="row">
-          <div class="col-6">
-            <div class="form-group">
-                <label asp-for="NewPassword" class="asterix control-label">@T["New password"]</label>
-                <input asp-for="NewPassword" class="form-control" aria-describedby="NewPassword-described-by" />
-                <div id="NewPassword-described-by" >
-                  <span asp-validation-for="NewPassword" class="text-danger"></span>
-                  <small class="form-text text-muted">@T["Your password must be at least 8 characters long and contain at least one uppercase letter, one lowercase letter, a number and a special character"]</small>
-                </div>
-            </div>
-            <div class="form-group">
-                <label asp-for="PasswordConfirmation" class="asterix control-label">@T["New password confirmation"]</label>
-                <input asp-for="PasswordConfirmation" class="form-control" aria-describedby="PasswordConfirmation-described-by" />
-                <span id="PasswordConfirmation-described-by" asp-validation-for="PasswordConfirmation" class="text-danger"></span>
-            </div>
-          </div>
-          <div class="col-6 d-flex align-items-center">
-            <small class="form-text text-secondary">@Html.Raw(T["Passwords must have:<br>• 8 characters or more<br>• an uppercase letter<br>• a lowercase letter<br>• a number<br>• a special character"])</small>
-          </div>
-        </div>
-      @await RenderSectionAsync("AfterResetPassword", required: false)
+<v-card class="px-4">
+  <v-card-title>@T["Reset password"]</v-card-title>
+  <v-card-subtitle>@T["Set a new password"]</v-card-subtitle>
+  <v-card-text>
+    <form id="form" asp-controller="ResetPassword" asp-action="ResetPassword" method="post">
+      @if (ViewData.ModelState.ErrorCount > 0)
+      {
+        <v-alert color="red" type="error">
+          <ul>
+            @foreach (var error in ViewData.ModelState.SelectMany(x => x.Value.Errors).Reverse())
+            {
+              <li>@error.ErrorMessage</li>
+            }
+          </ul>
+        </v-alert>
+      }
+      <v-row>
+        <v-col cols="12">
+          <v-text-field id="Email" type="email" name="Email" label="@T["Email"]" value="@email?.AttemptedValue" error-messages="@(email?.Errors.FirstOrDefault()?.ErrorMessage)"></v-text-field>
+          <v-row>
+            <v-col cols="auto">
+              <v-text-field id="NewPassword" type="password" name="NewPassword" label="@T["New password"]" value="@newPassword?.AttemptedValue" error-messages="@(newPassword?.Errors.FirstOrDefault()?.ErrorMessage)"></v-text-field>
+              <v-text-field id="PasswordConfirmation" type="password" name="PasswordConfirmation" label="@T["New password confirmation"]" value="@passwordConfirmation?.AttemptedValue" error-messages="@(passwordConfirmation?.Errors.FirstOrDefault()?.ErrorMessage)"></v-text-field>
```

Comment:
I may be wrong, but usually you don't want to pass the value back to password fields. Maybe asp.net does this by default ? not sure. In any case, I think we should remove all values for passwords as a security measure